docs/product/alerts/best-practices.mdx

```diff
@@ -37,6 +37,7 @@ Below, we describe best practices for setting alerts using the following frequen
 
 One way to keep alerts from becoming too noisy is to use filters, or “if” conditions, as part of your alert configuration. Below, we describe best practices for setting alerts using the following noise-reducing filtering options:
 
+- **Prioritize high priority issues**: If you're frequentrly notified about non-error or low priority issues, add the 'high priority' condition to your alert configuration. This will ensure that you only get alerts for high priority issues.
```

Comment:
```suggestion
- **Prioritize high priority issues**: If you're frequently notified about non-error or low priority issues, add the 'high priority' condition to your alert configuration. This will ensure that you only get alerts for high priority issues.
```